packages/appstore/src/components/options/options.tsx

```diff
@@ -0,0 +1,241 @@
+import React from 'react';
+import { useStores } from 'Stores';
+import { Text, StaticUrl, DesktopWrapper, MobileWrapper } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import PlatformLauncher from '../platform-launcher/index';
+import OptionsAccount from '../account/index';
+import AddOptions from '../add-options/index';
+import { isMobile, routes } from '@deriv/shared';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import { observer } from 'mobx-react-lite';
+import { getSortedAccountList } from '../../helpers';
+
+type TPlatformLauncherPropsArray = { icon: string; title: string; description: string; link_to: string }[];
+type TOptionsProps = {
+    platformlauncherprops: TPlatformLauncherPropsArray;
+};
+
+const Options: React.FunctionComponent<TOptionsProps & RouteComponentProps> = props => {
+    const { client, ui } = useStores();
+    const {
+        account_list,
+        accounts,
+        country_standpoint,
+        is_eu,
+        market_type,
+        server,
+        sub_account_type,
+        has_error,
+        platform,
+        is_dark_mode_on,
+        shortcode,
+        should_show_server_name,
+        resetVirtualBalance,
+    } = client;
+    const sortedAccountList = React.useMemo(
+        () => getSortedAccountList(account_list, accounts).filter(account => !account.is_virtual),
+        [account_list, accounts]
+    );
+    const [isActiveId, setIsActiveId] = React.useState(sortedAccountList[0].loginid);
+    const toggleAccountsDialog = ui.toggleAccountsDialog;
+    const resetBalance = async () => {
+        closeAccountsDialog();
+        resetVirtualBalance();
+    };
+    const closeAccountsDialog = () => {
+        toggleAccountsDialog(false);
+    };
+
+    const doDeposit = () => {
+        props.history.push(routes.cashier_deposit);
+    };
+
+    return (
+        <div className={`options-container ${!client.has_any_real_account ? 'options-container-app-launcher' : ''}`}>
```

Comment:
since `client.has_any_real_account` used many times, better to define a variable for that and reusing it across the file
```suggestion
       const has_any_real_account = client.has_any_real_account
```